apps/io-wallet-user-func/src/templates/wallet-instance-activation/index.html.ts

```diff
@@ -289,9 +282,6 @@ export default (faqLink: string, ioAppLink: string) => `
                                   </table>
                                 </td>
                               </tr>
-                              <div class="dark-img" style="mso-hide: all; display: none; padding: 10px 0px">
-                                <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJAAAACQCAMAAADQmBKKAAAARVBMVEUAAAAAc+YAcucAc+YAc+YAc+MAcuYAcN8Ac+cAc+QAcuYAcuYAc+QAdOYAc+YAc+gAcucAc+cAceQAdOYAcuYAcuQAc+ZZ8xRTAAAAFnRSTlMA3yDvv0CAEJ+QcN9Qz69vYF8wj49gsM2YIAAAA5ZJREFUeNrtnAlu4zAMRaldXuOk7b//UQdIMMEUY4cWacVG4HcA95mkqaVS6OTk5OTk5GNIoRkc4HzT0wFI0eGJiZZ25uLwC3OlXfnGf3zRjmTMkEmDPj4HitGEBVraBWuwgEu0BxGLRNoDAxwqRD1eEOj9NHhBQ+/H4wUDvR+HFzh6P3gJsXy+0OFSNuAFnlbx2Z99wAt6ej/JYZGO1vLRg+tyiDpLu9BigSvtRLP5J2b7Sx4Hc4+9M36M/W23SX4KjZ8rAucvN1WMGpHNZUZGtACdOvzCtVRO8GDx/dqsx3+UXEyitThD6aK4vyffDU1Q6fCYngQodHiypZoEg1JcS9VIIySMzyDtH54H5kY1+IacSJuTPHAkI2ug4+tgPkA+mA+QD+YDxIP5bLZPlwQ+Vfd8RmyGsxWWKio8qbHg0ZdRit7c575WUEA65v6i9eunK9/YGM/998W1dRPGLwFjSb9SJ4zfgL6U9KsJDPqGbR3THQQBUoUol3SHCVWIfJGmSgHitzPbknE4oBBj7wQw9PTEl3T0XCy0sld4dmfYMD1ILsSVSMmu+VRPKIqEfD0hLxGyqCeEJBDqawr9CIQy5nHR0q15Bv/BL6GU7zgskwVCA+Z5rNN/5mJiVrdVUy6UmHfzKiGkYqHAdNlWJxSKhVrmSZNOqC0WapgnRZ1QUyw0Yh7zyH4yOqGRESro04O9f9lYFgrgGQqEuPdzY84OSiFTLOTAoBNyxUKoK4RT6POFDlfUBgzv/uyHukJeN3TohfRDR1NBSDW4tnKhGxhE048gEHLMJF81QUsCISRmpqCZwpIRCFl6cAvMlsNA5UKNQOiH6fWqZdAkEGrWCl0FQkkg5NLKnFvRZkO5EL7WCXmSCLUCIcRVg8+VFxLn7AGemHwJIfSXzGRMIEQelfAkEwqoxFUgVDNEHUmFJlThKhaiDhXorFyoRwUiCYQqVlFHGqGAzbmqhKjBxgykE0pb17UVCNVMWiStELUVEqYSorFSC+KF6pdRoC2EyHYVCogXqm8USSBU0SiTQKiiUSaBUEWjTNsKka1zsNIobirFGsfhvOYkVi9NWxdK37KlVdgMCd7SIgmziG8Y6A94e+2h0Nipw1N0fojHlij5QByt/kSoXZs4fxV9vZHKmcaC6HC0jq85Hjt5vMDEVPCs/Kvm5CxfVip9qr3fNDI+JtJy6y+N93+vcw25+dn9ZzJOTk5OTk7exR910/wgI592uwAAAABJRU5ErkJggg==" width="36px" class="dark-img" style="display: none" />
-                              </div>
```

Comment:
why was it removed?

Reply:
Yes I understand the problem now (I didn't have the right context to understand the problem yet)! I'm on the right way to solve it